### PR TITLE
test_caching_multi_results: Remove unnecessary print

### PR DESCRIPTION
**Pull request by mariaandreicosmin**
Fixes https://github.com/coala/coala/issues/4071

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by jayvdb** [comment]
Please submit only one bug fix in each Pull Request.
You can keep using this Pull Request, by removing one of your commits (3ea2453 or 95c6cef).
Come onto http://coala.io/chat to get help removing the commit.

**Comment by arjunsinghy96** [comment]
Please run coala locally to pass all the tests. Also, as John said you can only summit one bugfix in one PR. You can remove either of them.

File: tests/misc/test_caching_multi_results/test.py [hunk 1]
```diff
@@ -1,2 +1 @@
-a = 3
-print(a)
+a = 3
```
Comment (andreidinu) [inline tests/misc/test_caching_multi_results/test.py]: hey, you just have to run coala locally. It will fix this kind of errors.

Comment (andreidinu) [inline tests/misc/test_caching_multi_results/test.py]: same here :)

**Comment by sims1253** [comment]
If you are not sure how to fix the comments you get from the bot, please come to our chat and ask us to help you.
coala.io/chat

**Comment by nityeshaga** [comment]
@mariaandreicosmin You still working on this PR?

**Comment by sils** [comment]
closing because inactivity.

**Pull request closed without merging**